### ISSUE TITLE
Set up TravisCI for this GitHub project

### ISSUE THREAD

**Issue by JeroenDeDauw** [SemanticMediaWiki/SemanticWatchlist#9]

**Comment by JeroenDeDauw** [comment]
@mwjames oh. And they run properly?

**Comment by mwjames** [comment]
> And they run properly?

Yes (for those classes that have a test) and as you can see the will also fail properly. Install via composer and `composer phpunit` will output:

```
Notice: Undefined index: egSWLEnableTopLink in ...\extensions\SemanticWatchlist\src\Setu
p.php on line 63

Call Stack:
    0.0155     160520   1. {main}() C:\xampp\htdocs\sandbox-import\tests\phpunit\phpunit.php:0
    0.0316     378656   2. require('C:\xampp\htdocs\sandbox-import\maintenance\doMaintenance.php') C:\xampp\htdocs\
sandbox-import\tests\phpunit\phpunit.php:199
    0.6941    3628288   3. require_once('C:\xampp\htdocs\sandbox-import\includes\Setup.php') C:\xampp\htdocs\sandbo
x-import\maintenance\doMaintenance.php:97
    1.8577    8162656   4. call_user_func() C:\xampp\htdocs\sandbox-import\includes\Setup.php:659
    1.8577    8162656   5. SWL\Setup->SWL\{closure}() C:\xampp\htdocs\sandbox-import\includes\Setup.php:659


Notice: Undefined index: egSWLEnableEmailNotify in C:\xampp\htdocs\sandbox-import\extensions\SemanticWatchlist\src\
Setup.php on line 64

Call Stack:
    0.0155     160520   1. {main}() C:\xampp\htdocs\sandbox-import\tests\phpunit\phpunit.php:0
    0.0316     378656   2. require('C:\xampp\htdocs\sandbox-import\maintenance\doMaintenance.php') C:\xampp\htdocs\
sandbox-import\tests\phpunit\phpunit.php:199
    0.6941    3628288   3. require_once('C:\xampp\htdocs\sandbox-import\includes\Setup.php') C:\xampp\htdocs\sandbo
x-import\maintenance\doMaintenance.php:97
    1.8577    8162656   4. call_user_func() C:\xampp\htdocs\sandbox-import\includes\Setup.php:659
    1.8577    8162656   5. SWL\Setup->SWL\{closure}() C:\xampp\htdocs\sandbox-import\includes\Setup.php:659


Notice: Undefined index: egSWLEnableEmailNotify in C:\xampp\htdocs\sandbox-import\extensions\SemanticWatchlist\src\
Setup.php on line 129

Call Stack:
    0.0155     160520   1. {main}() C:\xampp\htdocs\sandbox-import\tests\phpunit\phpunit.php:0
    0.0316     378656   2. require('...\maintenance\doMaintenance.php') C:\xampp\htdocs\
sandbox-import\tests\phpunit\phpunit.php:199
    0.6941    3628288   3. require_once('...\includes\Setup.php') C:\xampp\htdocs\sandbo
x-import\maintenance\doMaintenance.php:97
    1.8577    8162656   4. call_user_func() ...\includes\Setup.php:659
    1.8577    8162656   5. SWL\Setup->SWL\{closure}() ...\includes\Setup.php:659


SemanticMediaWiki 2.1 (sqlite) test autoloader ...

MediaWiki 1.24.1 root vendor autoloader ...

PHPUnit 4.4.2 by Sebastian Bergmann.

Configuration read from ...\extensions\SemanticWatchlist\phpunit.xml.dist

.E...ES..............

Time: 11.42 seconds, Memory: 20.75Mb

There were 2 errors:

1) SWL\Tests\ConfigurationIntegrityTest::testExtensionFunctionsRegistration
Undefined index: egSWLEnableTopLink

...\extensions\SemanticWatchlist\src\Setup.php:63
...\extensions\SemanticWatchlist\tests\phpunit\ConfigurationIntegrityTest.php:27

2) SWL\Tests\Integration\HookRegistrationIntegrationTest::testExtensionHookRegistration
Undefined index: egSWLEnableTopLink

...\extensions\SemanticWatchlist\src\Setup.php:63
...\extensions\SemanticWatchlist\tests\phpunit\Integration\HookRegistrationIntegrationTe
st.php:58 
```

**Comment by mwjames** [comment]
Fixed with #45
